docs/src/modules/components/AppNavDrawer.js

```diff
@@ -474,7 +474,7 @@ export default function AppNavDrawer(props) {
                         current: true,
                       }
                     : {
-                        href: `https://mui.com${languagePrefix}/components/data-grid/`,
+                        href: `https://v5.mui.com${languagePrefix}/components/data-grid/`,
                       }),
                 },
                 { text: 'v4', href: `https://v4.mui.com${languagePrefix}/components/data-grid/` },
```

Comment:
Eventually, we should be able to control product identifier in the MUI X repo to reduce the complexity of such a change.
We can add a slot prop to the `AppLayoutDocs` component.

Reply:
Cool, I did not know this PR exists 👍